cmake/modules/SwiftSharedCMakeConfig.cmake

```diff
@@ -268,16 +268,12 @@ endmacro()
 #     cmake variables.
 macro(swift_common_unified_build_config product)
   set(${product}_PATH_TO_CLANG_BUILD "${CMAKE_BINARY_DIR}")
-  set(CLANG_MAIN_INCLUDE_DIR "${LLVM_EXTERNAL_CLANG_SOURCE_DIR}/include")
-  set(CLANG_BUILD_INCLUDE_DIR "${CMAKE_BINARY_DIR}/tools/clang/include")
+  include_directories(${LLVM_EXTERNAL_CLANG_SOURCE_DIR}/include)
```

Comment:
why not `include_directories(${LLVM_EXTERNAL_CLANG_SOURCE_DIR}/include ${CMAKE_BINARY_DIR}/tools/clang/include)`

Reply:
I was doing replacement for a single entity at a time.  I can merge these two if you like. I don't think it makes much of a difference.